data/operators/amenity/charging_station.json

```diff
@@ -538,7 +538,8 @@
     {
       "displayName": "Chargefox",
       "id": "chargefox-943805",
-      "locationSet": {"include": ["au"]},
+      "locationSet": {"include": ["au", "nz"]},
+      "matchNames": ["chargefox"],
```

Comment:
Hello friends, welcome to name-suggestion-index, thank you for refine the locationSet.

However, the uppercase and lowercase letter's difference don't need to add as a matchNames, see wiki: https://github.com/osmlab/name-suggestion-index/wiki/Contributing#matchnamesmatchtags-optional

(It will be removed when auto-formatter program run so don't worry)

Reply:
thanks for clarifying that